### DISCUSSION TITLE
GPS and Compare icons on iPhone and other mobile devices

### DISCUSSION THREAD

**Discussion by imakihi** [TerriaJS/terriajs#5665] [Q&A] [answered]
Hi Terria community!

I realized that I could not show the GPS, Compare, and other tool icons when I open our terria instance with iPhone and other smartphone. I especially want to use GPS capability when I use those devices.

Can anyone tell me how to show those icons? 

I am using V8, next branch. I couldn't show those icons when I open the NSW terria neighter...

<img width="542" alt="2021-07-22_16-13-29" src="https://user-images.githubusercontent.com/4824436/126720506-d466da41-cff2-42e7-bcca-726275e58fe2.png">

Thank you!

Hiroo

**Comment by imakihi** [comment]
I checked this phenomenon with other browsers. Safari and Chrome didn't show icons but Edge showed them. I use iPhone XS, iOS14.
Thanks!
Hiroo

**Comment by imakihi** [comment, marked as answer]
So, I spent more time on this issue and realized that those buttons show up when I change the screen to the entire screen mode. So, I can see buttons at least on my iPhone. 

However, hotspot on those icons are bit off. When I tried to tap the GPS button, the measurement tool showed up.... But this is a bit different topic...

I am going to close this ticket.

Thank you!!

Hiroo

https://user-images.githubusercontent.com/4824436/126735837-a08ac508-9188-403e-ab9e-5ae37b779b15.mp4

**Reply by AnaBelgun** [reply]
Thanks @imakihi we do need to refresh the mobile icons on v8. Right now we'll focus on v8 migration and we'll look at mobile after.

**Reply by AnaBelgun** [reply]
Started here: https://github.com/TerriaJS/terriajs/issues/4548

**Reply by imakihi** [reply]
@AnaBelgun Thank you for your hard work!!
 I look forward to seeing the improvement on the mobile user interface!
Best regards.
Hiroo

**Reply by imakihi** [reply]
@AnaBelgun . I also realized that the mobile UI is not internationalized yet. I am sure you are aware of it but just in case.
Best,
Hiroo
<img width="247" alt="2021-07-24_8-55-17" src="https://user-images.githubusercontent.com/4824436/126874291-ea087d82-533e-4b44-b864-cde7bce2be58.png">
